.github/workflows/integration-tests.yml

```diff
@@ -229,7 +240,7 @@ jobs:
 
       - name: Upgrade @grafana packages using canary_version
         run: |
-          npm install --legacy-peer-deps $(echo $(cat package.json | jq -r --arg version $CANARY_VERSION '["@grafana/eslint-config", "@grafana/tsconfig", "@grafana/scenes", "@grafana/experimental"] as $blacklist | [(.devDependencies,.dependencies) | keys] | flatten | unique | map(select ( test("^@grafana") ) ) as $deps | $deps - $blacklist | map("\(.)@\($version)") | join(" ")') )
+          npm install --legacy-peer-deps $(echo $(cat package.json | jq -r --arg version $CANARY_VERSION '["@grafana/eslint-config", "@grafana/tsconfig", "@grafana/scenes", "@grafana/experimental", "@grafana/plugin-e2e"] as $blacklist | [(.devDependencies,.dependencies) | keys] | flatten | unique | map(select ( test("^@grafana") ) ) as $deps | $deps - $blacklist | map("\(.)@\($version)") | join(" ")') )
           npm install --force --save-optional @swc/core  @swc/core-linux-arm-gnueabihf @swc/core-linux-arm64-gnu @swc/core-linux-arm64-musl @swc/core-linux-x64-gnu @swc/core-linux-x64-musl
         working-directory: ${{ matrix.pluginDir }}
 
```

Comment:
Yeah, so I started to look into that but I'm not sure we want to use it here. Or at least we need to have a discussion about it in the squad.

So currently the integration tests run daily to verify that the examples are still working against the latest version of Grafana (which has changed). So with that setup it does not make any sense to re-run for older version every day.
But it might make sense to change this so we run it when a version changed regardless if it is the latest version or not and base it on the `grafanaDependency` flag for each example instead.

Reply:
Short answer is Yes! Agree - but I think we need to redesign this workflow a bit before we can do that so I will probably not cover that in this PR.